app/controllers/schools/partnerships_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Schools::PartnershipsController < Schools::BaseController
+  skip_after_action :verify_authorized
+  skip_after_action :verify_policy_scoped
```

Comment:
One day I will just cut them out from the base controller.

Reply:
I'm going to go through and replace `current_user.induction_coordinator_profile.schools.first` with `policy_scope(School)` I think